packages/terra-icon/package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "csvtojson": "^2.0.8",
     "jsdom": "^15.1.1",
-    "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/v1.41.2.tar.gz",
+    "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/refs/tags/v1.42.1.tar.gz",
```

Comment:
Seems odd to pull this from github instead of npm. @jmsv6d or @benbcai why don't we put this on NPM?

Reply:
Chalk this one up to, "This is how it was always done." I'd be interested to hear why this isn't on NPM as well.

Reply:
I think the reason is because one-cerner-style-icons is for Cerner use only and not intended to be consumed directly. It should only be consumed by terra-icon, which is published to NPM.